### PR TITLE
Add Basic Info buttons

### PR DESCRIPTION
**Pull request by AdamTReineke**
This allows the config to pass in values that can be displayed in buttons without panels (see `index.html`).

Also updates our TypeScript related dependencies and disable the Chrome test runner since it isn't working. The IE test runner still works, so tests still run.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +34,23 @@
                         {
                             panelConstructor: PerfToolbar.ResourceTimingsPanel,
                             config: {}
+                        },
+                        {
+                            panelConstructor: PerfToolbar.BasicInfoPanel,
+                            config: {
+                                buttons: [
+                                    {
+                                        emoji: "☁️",
```
Comment (JoshuaKGoldberg) [inline index.html]: 💖

File: src/panels/basic-info.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import { IPanel, IPanelConfig } from "../ipanel";
+import { PanelFrame } from "../panel-frame";
+
+export interface IBasicButton {
+    emoji: string;
+    title: string;
+    value(): string;
+}
+
+/** Describes the configuration options available for the network panel */
```
Comment (JoshuaKGoldberg) [inline src/panels/basic-info.ts]: > for the network panel

Nit: comment seems out of date

Comment (JoshuaKGoldberg) [reply src/panels/basic-info.ts]: Here and in a few other places in this file

File: src/panels/basic-info.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+export class BasicInfoPanel implements IPanel {
+    /** The settings for this panel. */
+    private readonly config: Required<IBasicInfoPanelConfig>;
+
+    /** Construct the panel. */
+    public constructor(_frame: PanelFrame, config?: IBasicInfoPanelConfig) {
+        this.config = { ...basicInfoPanelDefaultConfig, ...config };
+    }
+
+    /** Get buttons. */
```
Comment (JoshuaKGoldberg) [inline src/panels/basic-info.ts]: Nit: this JSDoc doesn't really help the reader, and in fact makes this class just a bit harder to read for taking up more space. Consider clarifying it or getting rid of it, maybe?

File: src/panels/basic-info.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                emoji: b.emoji,
+                getValue: () => b.value(),
+                getColor: () => "white",
+            }));
+        }
+
+        return out;
+    }
+
+    /** Do nothing. */
```
Comment (JoshuaKGoldberg) [inline src/panels/basic-info.ts]: Nit: incorrect copypasta?

File: karma.conf.js [hunk 5]
```diff
@@ -68,7 +68,7 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome', 'IE'],
+    browsers: [/*'Chrome', */'IE'],
```
Comment (JoshuaKGoldberg) [inline karma.conf.js]: Are you going to file an issue for this? Seems somewhat relevant :)

File: src/panels/basic-info.ts (hunk 3 above)
Comment (AdamTReineke) [reply src/panels/basic-info.ts]: The tslint requires it, so I'll make it a little better.

**Pull request merged**